src/plugins/saved_search/public/services/saved_searches/check_for_duplicate_title.ts

```diff
@@ -56,5 +56,5 @@ export const checkForDuplicateTitle = async ({
     return Promise.reject(new Error(`Saved search title already exists: ${title}`));
   }
 
-  return true;
+  return;
```

Comment:
The return type of this should be `void` not `boolean`:

https://github.com/elastic/kibana/blob/fa70f63aaa9f097f1143fc8685b2e1238d1985e0/packages/presentation/presentation_publishing/interfaces/has_library_transforms.ts#L12-L16